engines/decidim_hospitalet-surveys/config/locales/es.yml

```diff
@@ -52,6 +52,9 @@ es:
         success: Tu respuesta se ha enviado correctamente
       questions:
         age_group: Edad
+        authorize_lopd: Autorizo
+        authorize_lopd_hint: (Doy mi conformidad para que los datos personales aportados se incluyan en los ficheros de protección de datos de titularidad municipal, con la única finalidad que me sea enviada información relativa a las actividades relacionadas con la presente enquesta)
+
```

Comment:
Run `bundle exec i18n-tasks normalize` to remove this extra space. (or just remove it)